.gitlab/e2e/e2e.yml

```diff
@@ -471,6 +471,18 @@ new-e2e-ndm-snmp:
     TARGETS: ./tests/ndm/snmp
     TEAM: network-device-monitoring
 
+new-e2e-ha-agent:
+  extends: .new_e2e_template
+  rules:
+    - !reference [.on_ha_agent_or_e2e_changes]
+    - !reference [.manual]
+  needs:
+    - !reference [.needs_new_e2e_template]
+    - qa_agent
```

Comment:
If you are using a Host environment. You should depends on `deploy_deb_testing-a7_x64` instead.
Your test should work in the current situation because `qa_agent` deploys the Docker image that it created after the host packages. But to make it right you should depend `deploy_deb_testing-a7_x64`

Reply:
@KevinFairise2 good call, updated to extends `.new_e2e_template_needs_deb_x64` here: bf86b44bc0